samples/nrf9160/spm/pm.yml

```diff
@@ -1,2 +1,3 @@
 spm:
   placement: {before: [app]}
+  inside: [app_partition0]
```

Comment:
This change is immediately undone in the next patch. Why not squash them together?

Are there plans to document the `inside` feature in the partition manager user guide?

Reply:
Oh, good point, I will clean up the history.

The plan is to update the documentation, yes.

Reply:
FYI the documentation PR for PM is now in, so the needed doc changes can be added to this PR